zap/src/main/java/org/zaproxy/zap/view/StatusUI.java

```diff
@@ -39,7 +39,7 @@
      */
     public StatusUI(AddOn.Status status, String stringRepresentation) {
         if (status == null) {
-            this.status = AddOn.Status.unknown;
+            this.status = status = AddOn.Status.unknown;
```

Comment:
It could be changed the later one to `this.status` instead.

Reply:
You mean just change the order? `status = this.status = AddOn.Status.unknown`

Reply:
The one in `this.stringRepresentation = status.toString();`.